tests/SignedClosureTest.php

```diff
@@ -12,9 +12,14 @@
 
 class SignedClosureTest extends ClosureTest
 {
+    /**
+     * @expectedException \Opis\Closure\SecurityException
```

Comment:
Please don't do this with the annotation. Call the legacy `setExpectedException` method.